WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -33,13 +33,13 @@
           <tr>
             <td>
               <i class="flag f16 <%= competition.country.iso2.downcase %>"></i><%= link_to competition.cellName, competition_path(competition) %>
-              <br /><strong><%= competition.country.name_in(:en) %></strong>, <%= competition.cityName %>
```

Comment:
Is this intentional?

I'm fine with it but it looks more like a "localize admin page" than a "restore time_will_tell" thing :)